### PR TITLE
AWS update folder structure and mappings to 1.1.0

### PR DESCRIPTION
**Pull request by zschmerber**
Adding update for 1.1.0 folders and AWS 1.1.0 OCSF mappings

### CODE REVIEW

File: mappings/bloblang/AWS/CloudTrail/samples/account_change.ocsf
Comment (floydtree) [inline mappings/bloblang/AWS/CloudTrail/samples/account_change.ocsf:1]: Are you removing the transformed files intentionally? I think we should maintain .raw and .ocsf for samples for each version of mapping that is published. Similar to how we have the markdown mappings for AWS - https://github.com/ocsf/examples/tree/main/mappings/markdown/AWS

**Review by floydtree — CHANGES_REQUESTED** [review]

File: mappings/bloblang/AWS/CloudTrail/samples/account_change.ocsf
Comment (zschmerber) [reply mappings/bloblang/AWS/CloudTrail/samples/account_change.ocsf:1]: Yes, the thought was the person can use the Blobl to make a transformed file so why include it?

Comment (zschmerber) [reply mappings/bloblang/AWS/CloudTrail/samples/account_change.ocsf:1]: I can add them back if you think it would be helpful

Comment (floydtree) [reply mappings/bloblang/AWS/CloudTrail/samples/account_change.ocsf:1]: To help add context to the mappings themselves. Seeing a transformed record along with a raw record, helps in that regard. It helps in understanding the mappings as well i.e. x operation leads to such an output et al

Comment (zschmerber) [reply mappings/bloblang/AWS/CloudTrail/samples/account_change.ocsf:1]: Ok I will Update the PR thanks for the review.

**Review by Aniak5 — COMMENTED** [review]
I love the restructuring, it makes a lot of sense. `Account Change` is misspelled :D but otherwise looks great!

File: mappings/bloblang/AWS/route53/route53.yml
Comment (floydtree) [inline mappings/bloblang/AWS/route53/route53.yml:1]: Going back to the earlier comments (I see you added the structure and samples to CT, thank you!), r53 doesn't seem to have transformed samples. I think if we are adding samples, we should both raw and transformed, don't see only raw samples being as helpful to users.

**Review by floydtree — CHANGES_REQUESTED** [review]

**Review by floydtree — APPROVED** [review]

**Review by Aniak5 — APPROVED** [review]

**Review by mikeradka — APPROVED** [review]
The version folder helps, nice work

**Pull request merged**